demo/demo_director.py

```diff
@@ -53,6 +53,11 @@
 
 import atexit # to kill server process on exit()
 
+
+# Provided for consistency; used only by demo_primary currently.
```

Comment:
After reading this comment on its own, I was left wondering if setting `uptane.DEMO_MODE` to True causes special behavior in the reference implementation, or if demo code uses it as a flag to do alternative behavior of its own.

Reply:
Ah. True, I'll clarify.